include/mbedtls/aes.h

```diff
@@ -171,7 +177,9 @@ int mbedtls_aes_setkey_enc( mbedtls_aes_context *ctx, const unsigned char *key,
  * \brief          This function sets the decryption key.
  *
  * \param ctx      The AES context to which the key should be bound.
+ *                 It must be initialized.
  * \param key      The decryption key.
+ *                 This must be a readable buffer of size \p keybits bits.
```

Comment:
Minor: Bits instead of bits.

Reply:
I don't think we have to capitalize "bits". I think we have to capitalize "Bytes" because the abbreviation is a capital B so someone in the CC team thinks we should capitalize the word as well (which I don't think we should but I'm choosing my battles and this is not one I want to spend time one), but that argument clearly doesn't apply to "bits", quite the contrary.